gulpfile.js

```diff
@@ -863,11 +787,6 @@ exports.watch = initWatch;
 exports.watchCli = initWatchCli;
 exports.testlanguageservice = testlanguageservice;
 exports.onlinelearning = onlinelearning;
-exports.skillmap = skillmap;
-exports.authcode = authcode;
-exports.multiplayer = multiplayer;
-exports.kiosk = kiosk;
-exports.teacherTool = teacherTool;
 exports.tt = teacherTool;
```

Comment:
FWIW there's also `exports.tt` here (just a shortcut to teachertool), but I do actually use that on occasion when I only want to build teacher tool. Is there a reason we're removing these?

Reply:
they're still there! just being set inside the function defining the tasks

Reply:
Ahh, I see now. Makes sense!